myapp/spec/system/task_sorts_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe "TaskSorts", type: :system do
+  before do
+    @task1 = Task.create!(
```

Comment:
Since you're not using the `@task1` variable, you can simply create the task without assigning it to a variable.